src/components/CreateOrUpdateEvent.svelte

```diff
@@ -0,0 +1,176 @@
+<script>
+  import LocationInput from 'components/LocationInput.svelte'
+  import Uploader from 'components/Uploader.svelte'
+  import { throwError } from 'data/error.js'
+  import { events, update, create } from 'data/events.js'
+  import { me } from 'data/user.js'
+
+  import { format, addDays } from 'date-fns'
+  import { goto, params } from '@roxi/routify'
+
+  const tomorrow = format(addDays(Date.now(), 1), 'yyyy-MM-dd')
+
+  $: event = {}
+  $: existingEvent = $events.find(({ id }) => id === $params.eventId)
+  $: initializeUpdates(existingEvent)
+  $: isNew = !event.id
+
+  new Date().getDate
+
+  const parseDate = dateString => {
```

Comment:
Is this the same as `format(new Date(dateString), 'yyyy-MM-dd')`? Also, this shouldn't be needed. I see that the backend has a bug. It shouldn't return the ISO date/time format for these, but should give the yyyy-mm-dd format.